src/webextension/list/manage/components/account-summary-label.js

```diff
@@ -7,16 +7,17 @@ import React from "react";
 
 import styles from "./account-summary-label.css";
 
-export default function AccountSummaryLabel({email}) {
-  if (!email) {
+export default function AccountSummaryLabel({displayName, avatar}) {
+  if (!displayName) {
     return null;
   }
 
   return (
-    <span className={styles.accountSummary}>{email}</span>
+    <span className={styles.accountSummary}><img src={avatar} /> {displayName}</span>
```

Comment:
Nit: it might make more sense to separate this with a CSS margin than with a space character.

I also noticed that long display names take up a lot of space. Maybe this is something we should worry about in a followup though...